src/PrivateKey.cpp

```diff
@@ -256,7 +256,7 @@ Data PrivateKey::sign(const Data& digest, TWCurve curve, int (*canonicalChecker)
     return result;
 }
 
-Data PrivateKey::signAsDER(const Data& digest, TWCurve curve) const {
+Data PrivateKey::signAsDER(const Data& digest) const {
```

Comment:
@catenocrypt @hewigovens curve was not used, so I removed it just for transparency

Reply:
That's OK